src/headers.rs

```diff
@@ -141,14 +141,46 @@ impl Headers {
         self
     }
 
-    // TODO: time_zone
-    // TODO: language
+    /// Sets the timezone to be used when running the query, which by default is the timezone of the Presto engine.
+    /// Example: America/Los_Angeles
+    /// The `timezone` field must only contain visible ASCII characters (32-127);
```

Comment:
Separate from this PR, I think I should extract these repetitive warnings into a top-level warning.

Reply:
Well it's a tradeoff... I agree that the repetitive warning is a pain for maintainability. On the other hand, having the warning on top of each method will help make it visible to end-users through IDE tooltips and such. I would lean slightly towards making this fool-proof for the end users.